packages/core/src/domain/telemetry/telemetry.ts

```diff
@@ -87,7 +92,9 @@ export function startTelemetry(telemetryService: TelemetryService, configuration
         _dd: {
           format_version: 2 as const,
         },
-        telemetry: event as any, // https://github.com/microsoft/TypeScript/issues/48457
+        telemetry: combine(event, {
+          runtime_env: runtimeEnvInfo,
+        }) as any, // https://github.com/microsoft/TypeScript/issues/48457
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏the linked issue has been closed, should we do something about it?

Reply:
Yes I removed the `as any` and used the type assertion: https://github.com/DataDog/browser-sdk/blob/55cd64057c7245b979cdf4292b8fb5a582e9cdee/packages/core/src/domain/telemetry/telemetry.ts#L101